### PR TITLE
Add email field to form

### PR DESCRIPTION
**Pull request by jxrin**

### CODE REVIEW

File: pages/create/[id].js [hunk 1]
```diff
@@ -64,16 +70,30 @@ export default function Create() {
...
                                 className={styles.inputField}
                                 value={description}
                                 type="text"
-                                name="name"
+                                name="description"
                                 placeholder="Some words about what this card is for"
                                 required
                             />
+                            <p className={styles.inputName}>
+                                Event Organizer's Email
```
Comment (jevinsidhu) [inline pages/create/[id].js:86]: Maybe just "Organizer's Email"?

File: pages/create/[id].js [hunk 2]
```diff
@@ -64,16 +70,30 @@ export default function Create() {
...
+                            </p>
+                            <input
+                                onChange={handleEmail}
+                                className={styles.inputField}
+                                value={email}
+                                type="text"
```
Comment (jevinsidhu) [inline pages/create/[id].js:92]: use type="email"

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email

File: pages/create/[id].js [hunk 3]
```diff
@@ -64,16 +70,30 @@ export default function Create() {
...
+                                name="email"
+                                placeholder="john.smith@gmail.com"
```
Comment (jevinsidhu) [inline pages/create/[id].js:94]: would be cool to use more a more diverse name i think

maybe we can make an array and then randomly select one when the component mounts

**Review by jevinsidhu — APPROVED** [review]

File: pages/create/[id].js (hunk 3 above)
Comment (jxrin) [reply pages/create/[id].js:94]: Make an array in the same file? Kind of confused can you elaborate on this?

**Pull request merged**